tests/fake_device/raop.py

```diff
@@ -53,6 +100,9 @@ class FakeRaopState:
     def __init__(self):
         self.metadata = SimpleNamespace(title=None, artist=None, album=None)
         self.audio_packets: Dict[int, bytes] = {}  # seqo -> raw audio
+        self.password = None
```

Comment:
Please add type hints to these.

Reply:
Old python2 habits die hard. I will fix this.